02-loop.md

```diff
@@ -226,3 +226,37 @@ so we should always use it when we can.
 > Write a loop that takes a string,
 > and produces a new string with the characters in reverse order,
 > so `'Newton'` becomes `'notweN'`.
+
+> ## Ranking Olympians {.challenge}
+> 
+> Given a list of five men alpine skiing finalists at Sochi 2014 Winter Olympics:
+>
+> ~~~ {.python}
+> finalists = ['Viletta SUI', 'Kostelic CRO', 'Innerhofer ITA', 'Jansrud NOR', 'Zampa SVK']
+> ~~~
+> 
+> write a piece of code that prints ranked finalists:
+> 
+> ~~~ {.python}
+> 1 Viletta SUI
+> 2 Kostelic CRO
+> 3 Innerhofer ITA
+> 4 Jansrud NOR
+> 5 Zampa SVK
+> ~~~
+>
+> When working with collections such as lists and strings, it is common to use
+> the index of an element and the element inside the loop at the same time. To
+> make it easier for us, Python provides a handy `enumerate` object which
```

Comment:
Well, you can call `enumerate` outside a loop, but since it returns a generator (which isn't explained elsewhere), perhaps that's not a good idea...